src/DynamoCore/Graph/Connectors/ConnectorModel.cs

```diff
@@ -294,5 +294,15 @@ protected virtual void OnDeleted()
             var handler = Deleted;
             if (handler != null) handler();
         }
+
+        public override bool Equals(object obj)
+        {
+            return base.Equals(obj);
+        }
+
+        public override int GetHashCode()
+        {
+            return base.GetHashCode();   
+        }
```

Comment:
@mjkkirschner @aparajit-pratap do you guys remember the best practices for defining equality ?
The operators == and != are already overridden on this class

Found this https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/how-to-define-value-equality-for-a-type

Reply:
@Cristhian-Marin-asdk shouldn't the Equals and GetHashCode overridden methods align with the operator equality overrides? In other words. their implementations should be consistent.